docs/source/models/cause.rst

```diff
@@ -4,17 +4,119 @@
 Modeling Causes
 ===============
 
-.. todo::
-
-   Overview of modeling GBD causes.
+What is included in this chapter: models of causes that are represented as
+finite state Markov chain (this chapter will also explain what a finite state
+Markov chain is!)
+
+There are models of things that you might consider causes which are outside of
+the scope of this chapter, because they do not fit into the Markov approach.
+This chapter does not include “PAF of 1” causes, like protein-energy
+malnutrition (PEM).  It also does not include things that are sometimes
+considered a disease but are deemed “risk factors” by GBD, such as obesity.
+Unsafe water would not fit into the cause model paradigm either---it is a cause
+of diarrheal disease, but it is a risk factor in the GBD taxonomy.  Latent
+tuberculosis infection (LTBI) does fit into this chapter, but just barely.
 
 .. contents:
+   :local:
```

Comment:
@alibow This renders a table of contents.  The raw docs are like source code.  Many really cool and important features won't show up until we render them at https://vivarium-research.readthedocs.io/en/latest/

If you follow steps in the contribution guide, you can render them locally.  I always try to review docs on my own computer looking at the HTML page.  I'll show everyone how to do this on Friday.

Reply:
Thanks James! Sorry for the confusion.